### ISSUE TITLE
You need 0.9.41 version or above... Again!

### ISSUE THREAD

**Issue by padremayi** [AmatCoder/mednaffe#61]
```
Starting Mednaffe 0.8.6
Mednafen 09x configuration file found.
Error: Mednafen version is not compatible.
You need 0.9.41 version or above.
```

Mednafen version is 0.9.45.1.

Same problem with Mednaffe master version or Mednaffe 0.8.5

**Comment by AmatCoder** [comment]
Please, copy and paste here the output of mednafen emulator.

**Comment by padremayi** [comment]
For example this is the output playing Virtua Fighter 2:

```
$ ./mednafen '/media/padremayi/WINDOWS10/Users/padremayi/Downloads/Virtua Fighter 2 (Saturn) (E) v1.001/Virtua Fighter 2 v1.001 (1996)(Sega)(PAL)[!][MK-81014P-00623A].cue' 
Starting Mednafen 0.9.45.1
 Build information:
   Compiled with gcc 5.4.0 20160609
   Compiled against zlib 1.2.8, running with zlib 1.2.8(flags=0x000000a9)
   Compiled against SDL 1.2.15, running with SDL 1.2.15
   Running with libsndfile-1.0.25
 Base directory: /home/padremayi/.mednafen
 Emulation modules: nes snes gb gba pce lynx md pcfx ngp psx ss ssfplay vb wswan sms gg snes_faust pce_fast demo cdplay
 Loading settings from "/home/padremayi/.mednafen/mednafen-09x.cfg"...
  Loaded 5854 valid settings and 0 unknown settings.
 Initializing joysticks...
 Loading /media/padremayi/WINDOWS10/Users/padremayi/Downloads/Virtua Fighter 2 (Saturn) (E) v1.001/Virtua Fighter 2 v1.001 (1996)(Sega)(PAL)[!][MK-81014P-00623A].cue...
  Loading SBI file "/media/padremayi/WINDOWS10/Users/padremayi/Downloads/Virtua Fighter 2 (Saturn) (E) v1.001/Virtua Fighter 2 v1.001 (1996)(Sega)(PAL)[!][MK-81014P-00623A].sbi"...
   Error: Error opening file "/media/padremayi/WINDOWS10/Users/padremayi/Downloads/Virtua Fighter 2 (Saturn) (E) v1.001/Virtua Fighter 2 v1.001 (1996)(Sega)(PAL)[!][MK-81014P-00623A].sbi": No such file or directory

  CD 1 TOC:
   Disc Type: 0x00 (CD-DA or Mode 1)
   First Track:  1
   Last Track:  34
   Track  1, MSF: 00:02:00, LBA:      0  DATA
   Track  2, MSF: 05:51:47, LBA:  26222  AUDIO
   Track  3, MSF: 07:11:23, LBA:  32198  AUDIO
   Track  4, MSF: 08:44:69, LBA:  39219  AUDIO
   Track  5, MSF: 10:05:43, LBA:  45268  AUDIO
   Track  6, MSF: 11:26:42, LBA:  51342  AUDIO
   Track  7, MSF: 12:41:19, LBA:  56944  AUDIO
   Track  8, MSF: 14:19:66, LBA:  64341  AUDIO
   Track  9, MSF: 15:52:22, LBA:  71272  AUDIO
   Track 10, MSF: 17:23:65, LBA:  78140  AUDIO
   Track 11, MSF: 18:58:45, LBA:  85245  AUDIO
   Track 12, MSF: 20:13:62, LBA:  90887  AUDIO
   Track 13, MSF: 21:47:18, LBA:  97893  AUDIO
   Track 14, MSF: 23:02:42, LBA: 103542  AUDIO
   Track 15, MSF: 23:36:39, LBA: 106089  AUDIO
   Track 16, MSF: 23:42:39, LBA: 106539  AUDIO
   Track 17, MSF: 23:59:36, LBA: 107811  AUDIO
   Track 18, MSF: 24:48:15, LBA: 111465  AUDIO
   Track 19, MSF: 24:56:18, LBA: 112068  AUDIO
   Track 20, MSF: 25:05:29, LBA: 112754  AUDIO
   Track 21, MSF: 25:19:16, LBA: 113791  AUDIO
   Track 22, MSF: 28:20:73, LBA: 127423  AUDIO
   Track 23, MSF: 30:54:60, LBA: 138960  AUDIO
   Track 24, MSF: 33:55:17, LBA: 152492  AUDIO
   Track 25, MSF: 36:49:26, LBA: 165551  AUDIO
   Track 26, MSF: 39:23:29, LBA: 177104  AUDIO
   Track 27, MSF: 42:14:08, LBA: 189908  AUDIO
   Track 28, MSF: 45:51:46, LBA: 206221  AUDIO
   Track 29, MSF: 48:43:41, LBA: 219116  AUDIO
   Track 30, MSF: 51:02:30, LBA: 229530  AUDIO
   Track 31, MSF: 53:26:36, LBA: 240336  AUDIO
   Track 32, MSF: 55:56:24, LBA: 251574  AUDIO
   Track 33, MSF: 58:34:47, LBA: 263447  AUDIO
   Track 34, MSF: 59:57:15, LBA: 269640  AUDIO
   Leadout: 275286  AUDIO

 Using module: ss(Sega Saturn)
  Loading override settings from "/home/padremayi/.mednafen/ss.cfg"...
   Failed: Error opening file "/home/padremayi/.mednafen/ss.cfg": No such file or directory
  Loading override settings from "/home/padremayi/.mednafen/pgconfig/Virtua Fighter 2 v1.001 (1996)(Sega)(PAL)[!][MK-81014P-00623A].ss.cfg"...
   Failed: Error opening file "/home/padremayi/.mednafen/pgconfig/Virtua Fighter 2 v1.001 (1996)(Sega)(PAL)[!][MK-81014P-00623A].ss.cfg": No such file or directory

  Region: 0xc
  Cart: Backup Memory

  Displayed scanlines: [16,271]
  Correct Aspect Ratio: Enabled
  Show H Overscan: Enabled
  H Blend: Disabled

  Multitap on Saturn Port 1: Disabled
  Multitap on Saturn Port 2: Disabled

 Loading cheats from /home/padremayi/.mednafen/cheats/ss.cht...
  Error opening file "/home/padremayi/.mednafen/cheats/ss.cht": No such file or directory

 Initializing sound...
  Using "ALSA" audio driver with SexyAL's default device selection.
   Format: 16 bits(signed, 2 bytes, little-endian)
   Rate: 48000
   Channels: 2
   Buffer size: 1800 sample frames(37.500000 ms)
   Latency: 1800 sample frames(37.500000 ms)
   Period size: 60 sample frames(1.250000 ms)
 Initializing video...
  Video Driver: OpenGL
  Video Mode: 1095 x 768 x 32 bpp
  Shader: none
  Fullscreen: No
  Special Scaler: None
  Scanlines: Off
  Destination Rectangle: X=0, Y=0, W=1095, H=768
  OpenGL Implementation: Intel Open Source Technology Center Mesa DRI Intel(R) HD Graphics 620 (Kabylake GT2)  3.0 Mesa 17.0.7
  Checking extensions:
   GL_ARB_texture_non_power_of_two found.
   GL_ARB_sync found.
  Using non-power-of-2 sized textures.
  Checking maximum texture size...
   Apparently it is at least: 16384 x 16384
  Using GL_BGRA, GL_UNSIGNED_INT_8_8_8_8_REV for texture source data.
```

**Comment by AmatCoder** [comment]
Have you compiled mednafen emulator?

Be sure that **Bin** key points to your mednafen executable into _~/.config/mednaffe.conf_
Example:
```
# Version 0.8.6
# Do not edit this file!

[GUI]
Bin=/usr/bin/mednafen
[...]
```

**Comment by padremayi** [comment]
Yes, I compiled mednafen on Ubuntu 16.04. This evening I'll try and I'll give you a feedback.

I used your pop up window to select the mednafen executable, in this way I thought it must works

**Comment by padremayi** [comment]
Hi,
I don't have that configuration file, how is it possible?

**Comment by AmatCoder** [comment]
As I said in issue #52:
```
Have you an older mednafen emulator installed? Maybe from Ubuntu repositories?

Note that Ubuntu 16.04 has 0.9.38.7 version. You can search for executable in /usr/games
```

Mednaffe search for mednafen executable in $PATH if configuration file does not exist.

**Comment by padremayi** [comment]
I don't have any previous version of mednafen installed. As i said before I compiled the source code from scratch.

`padremayi@Ubuntu:/usr/games$ ls -l
total 660
-rwxr-xr-x 1 root root   1563 mag  6  2015 espdiff
-rwxr-xr-x 1 root root 106872 nov  2  2015 gnome-mahjongg
-rwxr-xr-x 1 root root 110984 feb 11  2016 gnome-mines
-rwxr-xr-x 1 root root 161120 ago 25  2016 gnome-sudoku
-rwxr-xr-x 1 root root 281848 nov 12  2015 sol`

Sorry but there isn't a wiki of your GUI, an example of your configuration file and where your application is searching for the mednafen executable.
You ask to user to manually select the emulator executable and it doesn't work. Maybe you have to recheck your code.

Thanks

**Comment by padremayi** [comment]
For any guy in the future, you have to wrote at the end of ~/.bashrc this line:

> export PATH=$PATH:path_to_mednafen_executable

When you will launch menaffe it doesn't found (again!!!) the executable but if you manually select the file it creates the configuration file in ~/.config

The configuration file has this contents.

> # Version 0.8.6
> # Do not edit this file!
> 
> [GUI]
> Bin=/home/ionic/Documents/saturn_test/mednafen/src/mednafen
> Folders=
> Last Folder=-1
> Tooltips=true
> RememberSize=true
> Width=1024
> Height=768
> Filter=0
> View Mode=0
> ActionLaunch=0
> Show Systems=true;true;true;true;true;true;true;true;true;true;true;true;true;true;true;

**Comment by AmatCoder** [comment]
> EDIT
The problem is very simple: the path for mednaffe executable can't has a space inside!!!

Yes, that is a bug. Thanks for find out the problem.

**Comment by AmatCoder** [comment]
Fixed with commit 8630356.

Rationale: **g_spawn_command_line_sync** function takes a command line instead of an argument vector, so it needs quotes.
